login-workflow/src/components/RegistrationWorkflow/RegistrationWorkflow.tsx

```diff
@@ -140,11 +143,13 @@ export const RegistrationWorkflow: React.FC<React.PropsWithChildren<Registration
             updateScreenData={updateScreenData}
             isInviteRegistration={isInviteRegistration}
         >
-            {showSuccessScreen
-                ? isAccountExist
-                    ? existingAccountSuccessScreen
-                    : successScreen
-                : screens[currentScreen]}
+            <ErrorManager {...errorDisplayConfig}>
```

Comment:
What's the intent of wrapping with an error manager when the screens already have an ErroManager component within them?

Reply:
I'll have to dig in again to recall this one, but when I was testing I'm pretty sure I needed to add this for the dialog to show up when a certain error was thrown.

Reply:
`completeRegistration()` perhaps?